set.nix

```diff
@@ -45,6 +45,10 @@ rec {
   */
   map = builtins.mapAttrs;
 
+  /* mapToList :: (key -> value -> value) -> set -> [value]
+  */
```

Comment:
I don't have a better name then this, though I think this one could be confusing.

Reply:
I think `mapToValues` is fine if you don't hate it, but *shrug*, up to you.

Reply:
yeah, can we do `mapToValues` instead? because the `toList` function returns a `[(Key, Value)]`.

Reply:
done!